src/Common.proto

```diff
@@ -33,6 +33,16 @@ message PeerId {
     bytes publicKey = 5; //20 bytes
 }
 
+/**
+* Provides a network error message, for when a node cant send/ build a valid response
+*/
+message ProtocolErrorMessageSigned {
+    bytes signature = 1;
+    PeerId peerId = 2;
+    bytes correlationId = 3;
+    int32 code = 4;
```

Comment:
Sorry being memory police here but do we need 4 bytes to represent this code? Can we not make this a byte, I don't think we'll have more than 256 error codes

Reply:
lol thats the field order @atlassanjay if we need this much space for error codes we're doing something wrong

Reply:
int32 = 4 bytes

Reply:
sorry I thought you were looking at field order, int32 in protobuffs is variable length, so not always 4 bytes https://developers.google.com/protocol-buffers/docs/proto3